docker/run_crontabber.sh

```diff
@@ -6,9 +6,27 @@
 
 # Runs crontabber.
 
+set -e
+
+# If this was kicked off via docker-compose, then it has a behavior
+# configuration already. If it wasn't, then we need to add behavior
+# configuration to the environment.
+if [[ -z "${CRONTABBER_BEHAVIOR}" ]];
+then
+    echo "Pulling in crontabber behavior configuration..."
+    CMDPREFIX="/app/bin/build_env.py /app/docker/config/crontabber.env"
+else
+    echo "Already have crontabber behavior configuration..."
+    CMDPREFIX=
+fi
+
+
 # Run crontabber sleeping 5 minutes between runs
-while true; do
-    python socorro/cron/crontabber_app.py
+while true
+do
+    # FIXME(willkg): We don't want crontabber to exit weird and then have that
+    # kill the container, but this is a lousy thing to do.
+    ${CMDPREFIX} python socorro/cron/crontabber_app.py || true
```

Comment:
If we change how this works per bug 1389571, then I think that'll fix this, too.